source/Target/CMakeLists.txt

```diff
@@ -1,3 +1,23 @@
+include_directories(../Plugins/Process/Utility)
+
+set(REFLECTION_LIB "")
+
+if(CMAKE_SYSTEM_NAME MATCHES "Linux")
+    set(REFLECTION_LIB "swiftReflection-linux-x86_64")
+endif()
+
+if(CMAKE_SYSTEM_NAME MATCHES "Darwin")
+    # libswiftReflection is placed in a subdirectory which depends
+    # on the host we're currently running on.
+    # FIXME: There must be a better way to get at libswiftReflection.a
+    string(REPLACE "-" ";" TRIPLE_LIST "${LLVM_HOST_TRIPLE}")
+    list(GET TRIPLE_LIST 0 ARCH)
+    list(GET TRIPLE_LIST 2 HOST_OS_VERSION)
+    string(REGEX MATCHALL "[a-z]+" HOST_OS "${HOST_OS_VERSION}")
+    set(REFLECTION_LIB "${REFLECTION_LIB}-${HOST_OS}-${ARCH}")
+    message("Building for non-linux host, setting reflection lib to ${REFLECTION_LIB}")
```

Comment:
We can probably remove that message, too. Either we print it unconditionally, or, just not at all.

Reply:
Ah ok, I'll see how testing goes and add that...